<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deletion</class>
 <widget class="QDialog" name="Deletion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: #3a3633;
}</string>
  </property>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>270</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="no_pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Нет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yes_pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
